ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/BeaconStateUtil.java

```diff
@@ -94,8 +108,8 @@ public static BeaconState get_initial_beacon_state(
                 UnsignedLong.valueOf(GENESIS_EPOCH)),
 
             // Validator registry
-            new ArrayList<Validator>(),
-            new ArrayList<UnsignedLong>(),
+            new ArrayList<>(),
```

Comment:
Keeping the generic type parameter explicitly defined in this case is nice bc it makes it easy to modify when the params change.

Reply:
IntelliJ doesn't like having them there so I just remove them to make IntelliJ happy 😂

Reply:
ok right on.  later on, i could add comments to make it easier to read.